reqhandler/fields.go

```diff
@@ -0,0 +1,33 @@
+package reqhandler
+
+type FieldName string
```

Comment:
Also, it looks like we are creating a a map of FieldName to []bytes only for the test.